src/main/java/ui/Ui.java

```diff
@@ -0,0 +1,91 @@
+package ui;
+
+import java.util.Scanner;
+
+import common.BotMessage;
+import common.TaskList;
+
+import tasks.Task;
+
+public class Ui {
+
+    Scanner scanner;
+
+    private final String header = """
+             \t  Hello! I'm Friday
+             \t  What can I do for you?""";
+
+    private final String ending = """
+             \t  Bye. Hope to see you again soon!""";
+
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    public String printList(TaskList list) {
+        String items = "";
+        if (list.isEmpty()) {
+            return "\t  There are currently no items in your list.";
+        }
+        for (int i = 0; i < list.size(); i++) {
+            String d = String.format("\t  %s.%s", i + 1, list.get(i));
+            if (i == list.size() - 1) {
+                items = items.concat(d);
+            } else {
+                items = items.concat(d + "\n");
+            }
+        }
+        return items;
+    }
+
+    public void showWelcome() {
+        BotMessage welcome = new BotMessage(header);
+        System.out.println(welcome);
+    }
+
+    public void showEnd() {
+        BotMessage end = new BotMessage(ending);
+        System.out.println(end);
+    }
+
+    public void showOutput(String str) {
+        BotMessage output = new BotMessage(str);
+        System.out.println(output);
+    }
+
+    public String readCommand() {
+        return this.scanner.nextLine();
+    }
+
+    public void listOutput(TaskList list) {
```

Comment:
Naming note: It's better to have function names as verbs, ex: `outputList` or `outputTaskAddition`.